### PR TITLE
Feature #182401190 – Reword main search form

### LINKED ISSUE #246: Metadata search: Hint to the users that conditions can be searched now
In the main search we currently see:

image

I would expect this to hint currently that not only Gene IDs or symbols can be searched for, but also conditions and cell types. So maybe "Gene ID, gene symbol, cell type or condition/factor" and possibly expand examples below (I would love to see for instance as example "epithelial cell of lung") which should take us to datasets that also have anatomograms. Maybe the curators can suggest more example to put below.

### PR DESCRIPTION
**Pull request by alfonsomunozpomer**
Remove label and add metadata search examples.

Solves #246 and https://github.com/ebi-gene-expression-group/atlas-web-single-cell/pull/262.

### CODE REVIEW

File: app/src/main/webapp/WEB-INF/jsp/tiles/views/home/search.jsp [hunk 1]
```diff
@@ -1,5 +1,7 @@
 <%--@elvariable id="resourcesVersion" type="String"--%>
 <%--@elvariable id="geneId" type="String"--%>
+<%--@elvariable id="resourcesVersion" type="String"--%>
```
Comment (ke4) [inline app/src/main/webapp/WEB-INF/jsp/tiles/views/home/search.jsp]: Are lines 3 and 4 the duplicate of lines 1 and 2?
If yes, then could we remove them, please?

File: app/src/main/webapp/WEB-INF/jsp/tiles/views/home/search.jsp [hunk 2]
```diff
@@ -15,41 +17,54 @@
...
-            text: 'MGI:98354 (MGI ID)',
-            url: '${pageContext.request.contextPath}/search?mgi_id=MGI:98354'
-          },
-          {
-            text: 'FBgn0004647 (FlyBase ID)',
-            url: '${pageContext.request.contextPath}/search?flybase_gene_id=FBgn0004647'
-          }
-        ]
-      }, 'search-form')
+            autocompleteLabel: ``,
```
Comment (ke4) [inline app/src/main/webapp/WEB-INF/jsp/tiles/views/home/search.jsp]: Should these be single quotes? I am not sure.

```suggestion
            autocompleteLabel: '',
```

**Review by ke4 — CHANGES_REQUESTED** [review]
Hey, I added 2 tiny suggestions to consider.
Could you have a look at them, please?
Otherwise, it looks fine.
Thanks

File: app/src/main/webapp/WEB-INF/jsp/tiles/views/home/search.jsp (hunk 2 above)
Comment (alfonsomunozpomer) [reply app/src/main/webapp/WEB-INF/jsp/tiles/views/home/search.jsp]: In our React packages we use [backtick strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) but in our JSPs we should use old-style JavaScript, with either double or single quotes. We chose the latter. So yes!

File: app/src/main/webapp/WEB-INF/jsp/tiles/views/home/search.jsp (hunk 1 above)
Comment (alfonsomunozpomer) [reply app/src/main/webapp/WEB-INF/jsp/tiles/views/home/search.jsp]: Oh, I missed those. Thanks!

**Review by ke4 — APPROVED** [review]
LGTM 👍

**Pull request merged**